packages/generator/src/entity/file.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('file', () => {
       element => element.kind === StructureKind.ImportDeclaration
     );
 
-    expect(imports.map(i => i.moduleSpecifier)).toIncludeSameMembers([
+    expect(imports.map(i => i.moduleSpecifier)).toEqual([
```

Comment:
[q] Are you sure this is correct on arrays? I remember that there were issues with deep equality sometimes. Did you test the negative case?

Reply:
yes I did a negative test. For string array it works as expected. The arrays need to have the same elements with `toEqual`. with the `toIncludeSameMembers` it can be a subset.